### ISSUE TITLE
Could not find mpf.exe

### ISSUE THREAD

**Issue by arthurkehrwald** [VisualPinball/VisualPinball.Engine.Mpf#15]
After fixing the compile errors described in [this issue](https://github.com/VisualPinball/VisualPinball.Engine.Mpf/issues/14), I ran into another error when attempting to get the machine description into Unity using the button on the MPF Game Logic engine in the inspector:
`VisualPinball: VisualPinball.Engine.Mpf.Unity.MpfGamelogicEngine|Unable to get machine description. Check maching config. Could not find mpf.exe!`

I am using Windows 11 with the latest version of this repo. I tried MPF version 0.55 with Python 3.7 as suggested in the, MPF version 0.56 with Python 3.9, and the latest MPF version 0.80 with Python 3.11. I was always able to run MPF on its own, but the error in Unity remains the same.

I am not sure where I would even get an MPF .exe file, since [official precompiled binaries](https://github.com/missionpinball/prepackaged-mpf-binaries/tree/main/latest) are only available for Linux and Mac.

**Comment by arthurkehrwald** [comment]
So I found mpf.exe in AppData\Local\Programs\Python\Python39\Scripts. I also looked at the code that is supposed to find it. It appends '\mpf.exe' to every path found in the PATH environment variable and checks if the file exists. Since my Python39/Scripts directory is in my PATH variable, it should find the executable this way, but for some reason it does not. I am currently trying to figure out how to debug the `Spawn` function in `MpfSpawner.cs` to understand why. So far, the debugger does not allow me to step into the `MpfApi.GetMachineDescription` function. Seems like it's compiled into a .dll, so that is probably why.
